modules/ROOT/pages/_partials/commons/common-scopes-collections-manage.adoc

```diff
@@ -61,6 +61,8 @@ NOTE: Scope and collection names are case sensitive.
 include::{root-partials}block_tabbed_code_example.adoc[]
 :param-tags!:
 
+NOTE: You cannot create an empty user-defined scope. 
+Scopes are an entity that contains collections, they are implicity created when a collection with documents associated to the provided scope is created.
```

Comment:
Might be worth using more specific terms here. In the Capella docs we refer to scopes as a "data container," but there might be something more appropriate from server docs.

```suggestion
Scopes are data containers that contain collections. 
```

Reply:
Will have a look!...It seems in Server that _collections_ are referred to as data containers whereas scopes are mechanisms to contain collections